test/framework/src/main/java/org/elasticsearch/index/mapper/KeywordFieldSyntheticSourceSupport.java

```diff
@@ -130,8 +130,8 @@ private void mapping(XContentBuilder b) throws IOException {
     public List<MapperTestCase.SyntheticSourceInvalidExample> invalidExample() throws IOException {
         return List.of(
             new MapperTestCase.SyntheticSourceInvalidExample(
-                equalTo("field [field] of type [keyword] doesn't support synthetic source because it declares a normalizer"),
-                b -> b.field("type", "keyword").field("normalizer", "lowercase")
+                equalTo("field [field] of type [keyword] doesn't support synthetic source because it declares copy_to"),
```

Comment:
You can remove this, `copy_to` is unconditionally tested for all mappers in the base class.